ibm/service/power/resource_ibm_pi_volume.go

```diff
@@ -266,6 +278,13 @@ func resourceIBMPIVolumeCreate(ctx context.Context, d *schema.ResourceData, meta
 
 	}
 
+	if v, ok := d.GetOk(Arg_UserTags); ok {
+		userTags := flex.ExpandStringList(v.([]interface{}))
+		if len(userTags) > 0 {
+			body.UserTags = userTags
+		}
+	}
+
```

Comment:
Could you check for `Len(v.([]interface{}) > 0` first and then proceed with the rest?